orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -381,9 +391,46 @@ def forward(self, input1, input2, input3):
     y = torch.randn(N, D_in, device=device)
     z = torch.randn(N, D_in, device=device)
 
-    with pytest.raises(NotImplementedError) as not_implemented_error:
-        model(x, y, z)
-    assert str(not_implemented_error.value) == 'Dictionaries are not supported as output yet'
+    output = model(x, y, z)
+    assert isinstance(output, return_type)
+    assert 'loss' in output and 'logits' in output and 'hidden_states' in output
+
+@pytest.mark.parametrize("device", ['cpu', 'cuda'])
+def test_named_tuple_return_value_module(device):
```

Comment:
How about some tests that handle unsupported types?

Reply:
I'll add a unit test for that.